setup.py

```diff
@@ -0,0 +1,24 @@
+from time import time
+
+from setuptools import setup
+from setuptools_scm.version import ScmVersion
+
+
+def get_dev_timestamp(version: ScmVersion) -> str:
+    """Returns a new distribution version string.
```

Comment:
```suggestion
    """Return a new distribution version string.
```

Obviously a nitpick. Some people, e.g. [0]. find it important to keep the respective first line of a doc string in imperative mood.

PEP 257 [1] says the following:
> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

I don't have a strong opinion myself, just wanted to make sure you're aware of the context.

[0] https://stackoverflow.com/questions/59902102/why-is-imperative-mood-important-for-docstrings
[1] https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
Thanks, I usually pay attention to details like this. I might have forgotten this convention.